src/server/controllers/tasks.js

```diff
@@ -74,21 +74,10 @@ export class TaskManager {
       }
     });
   }
-
   insertTask() {
     const newTask = new Task(this.req.body);
     newTask.save((error, tasks) => {
-      console.log('error');
-      if (error) {
-        console.log('error');
-        const response = error.message;
-        this.res.json({
-          error: response,
-        });
-      } else {
-        console.log('error');
-        this.res.json(tasks);
-      }
```

Comment:
You deleted vital error handling... please add it again. My comment referred ONLY to the console logs.